API/Controllers/ElectricityController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using API.Data;
+using API.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace trackingapi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ElectricityController : ControllerBase
+    {
+        private readonly IssueDbContext _context;       //Allow controller to access the database
+
+        public ElectricityController(IssueDbContext context) => _context = context;
+
+
+
+        [HttpGet] //All created units
+        public async Task<IEnumerable<Issue>> Get()
+            => await _context.Issues.ToListAsync();
+
+        [HttpGet("{id}")] //Get unit by it's id
+        [ProducesResponseType(typeof(Issue), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> GetById(int id)
+        {
+            var issue = await _context.Issues.FindAsync(id);
+            return issue == null ? NotFound() : Ok(issue);
+        }
+
```

Comment:
Why need? From 32 line

Reply:
Created for testing purposes, will be removed later.

Reply:
Works for me